### PR TITLE
[Improvements] improved readme

### PR DESCRIPTION
**Pull request by anurov-prykhodko**
I added pictures for effect and a description of the functionality that has already been implemented.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,9 +18,16 @@ A personal assistant with a command line interface.
...
+
+- Ability to add text notes.
+- Search, edit, and delete notes.
+- You can add "tags" to your notes, keywords that describe the topic and subject of the record.
+- Search and sort notes by keywords (tags).
+
 ## Installation
 
-Muad'Dib requires [Node.js](https://nodejs.org/) v10+ to run.
+Muad'Dib requires [Node.js](https://nodejs.org/) v10+ & [Python](https://www.python.org/) to run.
```
Comment (OleinikovArtem) [inline README.md]: Do we really need node.js? For What?

File: README.md [hunk 2]
```diff
@@ -37,18 +44,38 @@ Instructions on how to use them are provided below.
...
 | exit | Exits assistant |
 | close | Alias for Exit command |
 
+## Aditional features
+
+Muad'Dib can store and manage your notes by adding "tags" to your notes + searching notes by these "tags".
+
+| Сommands for notes | Response |
+| ------ | ------ |
+| add-note | Adds new note. Required arguments: title, content. Optional argument: tags |
```
Comment (OleinikovArtem) [inline README.md]: So for note we don't use arguments, but use a list of questions for each field. 
for `edit-note` too

**Review by OleinikovArtem — DISMISSED** [review]

File: README.md [hunk 3]
```diff
@@ -37,18 +44,38 @@ Instructions on how to use them are provided below.
 | ------ | ------ |
 | help | Shows all available commands |
 | hello | Greeting command |
-| add-contact | Adds new contact. Required arguments: name, phone (10 digits) |
+| add-contact | Adds new contact. Required arguments: name, phone (10 digits). Optional argument: email, birthday |
```
Comment (thebedcoder) [inline README.md]: we don't have arguments for add-contact command, but we have subset of questions to fulfil. Just like in add-note command

File: README.md [hunk 4]
```diff
@@ -37,18 +44,38 @@ Instructions on how to use them are provided below.
...
 | change-contact | Changes existing contact. Required arguments: name, old phone, new phone |
```
Comment (thebedcoder) [inline README.md]: and the same for change-contact

**Review by thebedcoder — CHANGES_REQUESTED** [review]
Great job! It looks awesome!
2 minor comments

**Review by thebedcoder — APPROVED** [review]

**Pull request merged**